common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -283,6 +305,14 @@ def test_deferred_verification_enrollment(self):
         # Submit payment
         self.fake_payment_page.submit_payment()
 
+        # Expect enrollment activated event
+        assert_event_emitted_num_times(
```

Comment:
Huh, in that case I'd leave this check where it is (as well as move the check on line 246, as I see you've done). I must be misunderstanding something about the time at which the events are emitted.